lib/Conversion/LLHDToLLVM/LLHDToLLVM.cpp

```diff
@@ -0,0 +1,1427 @@
+#include "circt/Conversion/LLHDToLLVM/LLHDToLLVM.h"
+#include "circt/Dialect/LLHD/IR/LLHDDialect.h"
+#include "circt/Dialect/LLHD/IR/LLHDOps.h"
+
+#include "mlir/Conversion/StandardToLLVM/ConvertStandardToLLVM.h"
+#include "mlir/Conversion/StandardToLLVM/ConvertStandardToLLVMPass.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/IR/BlockAndValueMapping.h"
+#include "mlir/IR/Dominance.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir {
+namespace llhd {
+#define GEN_PASS_CLASSES
+#include "circt/Conversion/LLHDToLLVM/Passes.h.inc"
+} // namespace llhd
+} // namespace mlir
+
+using namespace mlir;
+using namespace mlir::llhd;
+
+namespace {
```

Comment:
Please use `static` for functions like getGlobalString etc.

Anon namespaces should only be used for structs/classes (like `EntityOpConversion`) and should be tightly wrapped around them to make it easier to understand visibility at a glance.